### PR TITLE
cosmetic changes

### PR DESCRIPTION
**Pull request by rogpeppe**
Use the "bakery" identifier for the macaroonbakery package everywhere.

Use all-args-on-separate-lines style when we want to split
a function onto multiple lines, as it's less sensitive to
automated changes that change the length of the initial
function call.

Split multiple import-from statements across multiple lines
and sort the imported identifiers.

### CODE REVIEW

File: macaroonbakery/httpbakery/error.py [hunk 1]
```diff
@@ -78,21 +77,21 @@ def request_version(req_headers):
     version is used, which is OK because versions are backwardly compatible.
 
     @param req_headers: the request headers as a dict.
-    @return: bakery protocol version (for example macaroonbakery.BAKERY_V1)
+    @return: bakery protocol version (for example bakery.BAKERY_V1)
```
Comment (ricardokirkner) [inline macaroonbakery/httpbakery/error.py]: I think I'd keep the original name in the docstring (eg, macaroonbakery.BAKERY_V1) as I'm not sure the alias is visible when viewing the docs directly.

File: macaroonbakery/httpbakery/keyring.py [hunk 2]
```diff
@@ -3,13 +3,12 @@
...
-import macaroonbakery
+import macaroonbakery as bakery
 from macaroonbakery.httpbakery.error import BAKERY_PROTOCOL_HEADER
-from macaroonbakery import LATEST_BAKERY_VERSION
 
 
-class ThirdPartyLocator(macaroonbakery.ThirdPartyLocator):
-    ''' Implements macaroonbakery.ThirdPartyLocator by first looking in the
+class ThirdPartyLocator(bakery.ThirdPartyLocator):
+    ''' Implements bakery.ThirdPartyLocator by first looking in the
```
Comment (ricardokirkner) [inline macaroonbakery/httpbakery/keyring.py]: ditto

File: macaroonbakery/httpbakery/error.py (hunk 1 above)
Comment (rogpeppe) [reply macaroonbakery/httpbakery/error.py]: good point. done.

File: macaroonbakery/httpbakery/keyring.py (hunk 2 above)
Comment (rogpeppe) [reply macaroonbakery/httpbakery/keyring.py]: done

**Pull request merged**